src/assets/scss/_range-input.scss

```diff
@@ -0,0 +1,81 @@
+.range-input-container {
+    display: flex;
+    flex-direction: row;
+    align-items: end;
+    width: 100%;
+    flex-wrap: wrap;
+    margin-top: -5px;
+
+    .range-input {
+        width: 40%;
+        display: flex;
+        flex-direction: column;
+        margin-top: 5px;
+
+        margin-right: 5px;
+        flex-shrink: 0;
+        max-width: 160px;
+
+        &.numeric-width,
+        &.date-width,
+        &.time-width {
+            min-width: 140px;
+        }
+
+        label {
+            font-weight: 700;
+        }
+
+        .chaise-input-control {
+
+            &.has-feedback {
+                height: unset;
+                min-height: auto;
+            }
+
+            .range-input-clear {
+                top: 3px;
+            }
+        }
+
+        .range-input-field {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            position: relative;
+        }
+
+        .range-input-datetime {
+            flex-direction: column;
+
+            .chaise-input-control:first-child {
+                margin-bottom: 5px;
+            }
+        }
+
+        #range-from-val-date,
```

Comment:
Why are you writing rules based on `id`? You should do this with `class` instead.

Reply:
fixed